// react-search-autocomplete/dist/SearchInput/SearchInput.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = SearchInput;

var _react = _interopRequireDefault(require("react"));

var _propTypes = _interopRequireDefault(require("prop-types"));

var _StyledSearchInput = require("./StyledSearchInput");

var _SearchIcon = require("./../icons/SearchIcon");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function SearchInput(props) {
  var searchString = props.searchString,
      setSearchString = props.setSearchString,
      autoFocus = props.autoFocus,
      onBlur = props.onBlur,
      onFocus = props.onFocus,
      placeholder = props.placeholder,
      showIcon = props.showIcon;
  return /*#__PURE__*/_react["default"].createElement(_StyledSearchInput.StyledSearchInput, null, showIcon && /*#__PURE__*/_react["default"].createElement(_SearchIcon.SearchIcon, null), /*#__PURE__*/_react["default"].createElement("input", {
    spellCheck: false,
    value: searchString,
    onChange: setSearchString,
    onBlur: onBlur,
    onFocus: onFocus,
    placeholder: placeholder,
    autoFocus: autoFocus
  }));
}

SearchInput.defaultProps = {
  showIcon: true
};
SearchInput.propTypes = {
  searchString: _propTypes["default"].string.isRequired,
  setSearchString: _propTypes["default"].func.isRequired,
  autoFocus: _propTypes["default"].bool,
  onBlur: _propTypes["default"].func.isRequired,
  onFocus: _propTypes["default"].func,
  placeholder: _propTypes["default"].string,
  showIcon: _propTypes["default"].bool
};
